index.html

```diff
@@ -0,0 +1,1305 @@
+        <div class="mt-5">
+          <h5 class="text-muted">LATEST NEWS</h5>
+          <div class="media latest pb-3 mt-3">
+            <img src="images/deck_img.jpg" class="debate img-fluid d-flex mr-3" alt="newsweek-img">
+
+            <div class="media-body">
+              <h6 class="media-title font-weight-bold">More Aggressive' Coronavirus Strain May Have Evolved—But Don't
+                Worry</h6>
+
+            </div>
+          </div>
+        </div>
+        <div class="media  latest pb-3 mt-3">
+          <img src="images/deck_img.jpg" class="debate img-fluid d-flex mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h6 class="media-title font-weight-bold">More Aggressive' Coronavirus Strain May Have Evolved—But Don't
+              Worry</h6>
+
+          </div>
+        </div>
+        <div class="media latest  pb-3 mt-3">
+          <img src="images/deck_img.jpg" class="debate img-fluid d-flex mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h6 class="media-title font-weight-bold">More Aggressive' Coronavirus Strain May Have Evolved—But Don't
+              Worry</h6>
+
+          </div>
+        </div>
+        <div class="media latest pb-3 mt-3">
+          <img src="images/deck_img.jpg" class="debate img-fluid d-flex mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h6 class="media-title font-weight-bold">More Aggressive' Coronavirus Strain May Have Evolved—But Don't
+              Worry</h6>
+          </div>
+        </div>
+        <div class="media latest pb-3 mt-3">
+          <img src="images/deck_img.jpg" class="debate img-fluid d-flex mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h6 class="media-title font-weight-bold">More Aggressive' Coronavirus Strain May Have Evolved—But Don't
+              Worry</h6>
+          </div>
+        </div>
+        <div class="media latest latest-bottom pb-3 mt-3">
+          <img src="images/deck_img.jpg" class="debate img-fluid d-flex mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h6 class="media-title font-weight-bold">More Aggressive' Coronavirus Strain May Have Evolved—But Don't
+              Worry</h6>
+          </div>
+        </div>
+        <div class=" col-md-12 m-md-auto m-sm-auto news-letter sticky">
+          <div class="d-flex mt-4">
+            <span class="icon-n mt-4 ml-3"></span>
+            <p class="mt-3 ml-2 font-weight-bold para">SIGN UP FOR OUR NEWSLETTER</p>
+          </div>
+          <button class="form-control signup mt-3 ml-3"> SIGN UP</button>
+          <h6 class="mt-3 ml-3">Update your preferences »</h6>
+        </div>
+      </div>
+    </div>
+
+    <div class="row  pl-lg-4 pr-lg-4 bg-white">
+      <div class="col-12 mt-4">
+        <div class="d-flex magazine">
+          <h5 class="m-auto d-flex pb-2"><span class="icon-n mr-3"></span>IN THE MAGAZINE</h5>
+        </div>
+      </div>
+    </div>
```

Comment:
The article images in this section are wider due to the general width of your page. Please make sure that when you fix the body, this section reflects those changes, too.

In the medium breakpoint, you can see the same problem: 
Yours:
![Screenshot from 2020-03-14 18-34-44](https://user-images.githubusercontent.com/51926937/76692890-9b617c80-6622-11ea-8d9e-fe7fc8d2ac1f.png)

Original:
![Screenshot from 2020-03-14 18-34-36](https://user-images.githubusercontent.com/51926937/76692891-9ef50380-6622-11ea-9ac2-f2ef690552f8.png)